templates/default/varnish.erb

```diff
@@ -110,6 +110,7 @@ DAEMON_OPTS="-a ${VARNISH_LISTEN_ADDRESS}:${VARNISH_LISTEN_PORT} \
              <% if node['varnish']['GeoIP_enabled'] %>
                  -p 'cc_command=exec cc -fpic -shared -Wl,-x -L/usr/local/lib -lGeoIP -o %o %s' \
              <% end %>
+             <%= (node['varnish']['custom_parameters'] || []).map { |p| "-p '#{p}'" }.join("\n") %>
```

Comment:
I wonder if there is a corner case for this, but http://apidock.com/ruby/Shellwords/shellescape might be used instead of `'#{p}'`